CodeGeneration/Sources/SyntaxSupport/Traits.swift

```diff
@@ -66,6 +66,13 @@ public let TRAITS: [Trait] = [
       Child(name: "memberBlock", kind: .node(kind: .memberBlock)),
     ]
   ),
+  Trait(
+    traitName: "EffectfulExpr",
+    documentation: "Syntax trait for effectful expressions, such as `try` and `await`.",
+    children: [
+      Child(name: "expression", kind: .node(kind: .expr))
+    ]
+  ),
```

Comment:
We should probably also expose the `try` or `await` keyword, similar to https://github.com/apple/swift-syntax/pull/2539

Reply:
I've incorporated a `keyword` property into the trait.

However, I have some reservations regarding this API, particularly because using the `try` keyword without accompanying it with either a question mark or an exclamation mark might lead to confusion.

Reply:
Perhaps you could expose an `effectKeyword` property (mirroring `tryKeyword` and `awaitKeyword`) as well as a `questionOrExclamationMark` property that's just always `nil` for `AwaitExprSyntax`?

(In the future, we could conceivably add some sort of `await?` or `await!` syntax and then the property would be automagically available. And no, I don't know what such syntax would _do_ since it's purely hypothetical.)